R/session_check_reauth.R

```diff
@@ -50,7 +50,8 @@ session_check_reauth = function(username, password, session, ...){
 	}else{ #if we have passed nothing, use cached session
```

Comment:
at https://github.com/jread-usgs/sbtools/blob/master/R/session_check_reauth.R#L34, reauthentication only happens if the user name has changed. what about if it expires? would it make sense to add a session_validate to that if block?